app/views/sufia/batch_uploads/_form_files.html.erb

```diff
@@ -98,10 +98,17 @@
                   <div><span class="label label-danger">Error</span> {%=file.error%}</div>
               {% } %}
               <span class="size">{%=o.formatFileSize(file.size)%}</span>
-              <button class="btn btn-warning cancel pull-right">
-                  <i class="glyphicon glyphicon-ban-circle" aria-hidden="true"></i>
-                  <span>Cancel</span>
-              </button>
+              {% if (file.deleteUrl) { %}
+                <button class="btn btn-danger delete pull-right" data-type="{%=file.deleteType%}" data-url="{%=file.deleteUrl%}"{% if (file.deleteWithCredentials) { %} data-xhr-fields='{"withCredentials":true}'{% } %}>
+                    <i class="glyphicon glyphicon-trash"></i>
+                    <span>Delete</span>
+                </button>
+              {% } else { %}
+                <button class="btn btn-warning cancel pull-right">
+                    <i class="glyphicon glyphicon-ban-circle" aria-hidden="true"></i>
+                    <span>Cancel</span>
```

Comment:
My preference is to expose only the button that triggers a back-end delete, and I don't care whether we call it Delete or Cancel. I defer to folks like @mtribone @ggeisler @aeschylus.